agogosml/agogosml/reader/input_reader_factory.py

```diff
@@ -13,9 +13,10 @@ class InputReaderFactory:
     """Factory and instance resolving for input reader"""
 
     @staticmethod
-    def create(config: dict):
+    def create(config: dict, streaming_client=None):
         """Create a new instance of InputReader
 
+        :param streaming_client:
```

Comment:
The order of the params is reversed to the signature. Please add more details in the description